parametric/test_otel_span_methods.py

```diff
@@ -0,0 +1,222 @@
+import time
+
+import pytest
+
+from parametric.spec.otel_trace import OTEL_UNSET_CODE, OTEL_ERROR_CODE, OTEL_OK_CODE
+from parametric.spec.otel_trace import OtelSpan
+from parametric.spec.otel_trace import SK_PRODUCER
+from parametric.spec.trace import find_span
+from parametric.spec.trace import find_trace_by_root
+from parametric.utils.test_agent import get_span
+
+
+@pytest.mark.skip_library("dotnet", "Not implemented")
+@pytest.mark.skip_library("nodejs", "Not implemented")
+@pytest.mark.skip_library("python", "Not implemented")
+@pytest.mark.skip_library("java", "Not implemented")
+def test_otel_start_span(test_agent, test_library):
+    """
+        - Start/end a span with start and end options
+    """
+
+    with test_library:
+        duration: int = 6789
+        start_time: int = 12345
+        with test_library.start_otel_span(
+            "operation",
+            span_kind=SK_PRODUCER,
+            timestamp=start_time,
+            new_root=True,
+            attributes={"start_attr_key": "start_attr_val"},
+        ) as parent:
+            parent.otel_end_span(timestamp=start_time + duration)
+
+    root_span = get_span(test_agent)
+    assert root_span["name"] == "operation"
+    assert root_span["resource"] == "operation"
```

Comment:
can we set the name and the resource different things, to make sure they're being set correctly, or is this just how the OTel API works?

Reply:
This is technically how _Datadog_ works, if there's no resource specified with Datadog StartSpanOptions. So, simple OTel span would have same name and resource